pkg/backend/aio.go

```diff
@@ -45,8 +45,12 @@ func (s *Server) CreateAioVolume(_ context.Context, in *pb.CreateAioVolumeReques
 	}
 	in.AioVolume.Name = utils.ResourceIDToVolumeName(resourceID)
 	// idempotent API when called with same key, should return same object
-	volume, ok := s.Volumes.AioVolumes[in.AioVolume.Name]
-	if ok {
+	volume := new(pb.AioVolume)
+	found, err := s.store.Get(in.AioVolume.Name, volume)
+	if err != nil {
+		return nil, err
```

Comment:
I'd explicitly set gRPC error code as `Internal` in all storage related operations by means of` status.Errof(codes.Internal,...)`